src/SFA.DAS.EAS.Employer_Financial.Database/StoredProcedures/GetAllPaymentsForLastYearByAccountId.sql

```diff
@@ -0,0 +1,8 @@
+﻿CREATE PROCEDURE [employer_financial].[GetAllPaymentsForLastYearByAccountId]
+	@AccountId BIGINT
+AS
+	SELECT SUM(Amount)*-1 AS TotalSpend 
+	FROM employer_financial.TransactionLine 
+	WHERE TransactionDate >= DATEADD(YEAR,-1,GETDATE())
```

Comment:
This code will take into consideration the date and time meaning that it will get everything from DD-MM-YYYY hh:mm:ss   - DD-MM-YYYY+1 hh:mm:ss.

Also it does not account for daylight saving which is fine if we save our transactions as UTC

Reply:
Now with less time! goes from midnight from the year before to current time current year